src/main/java/dev/jbang/dependencies/JitPackUtil.java

```diff
@@ -11,17 +11,17 @@
 
 public class JitPackUtil {
 	private static final Pattern GITHUB_TREE_PATTERN = Pattern.compile(
-			"^https?://github.com/(.+?)/(.+?)(/tree/(.+?)(/(.+?))?)?/?$");
+			"^https?://github.com/([^/]+?)/([^/]+?)(/tree/([^/]+?)(/(.+?))?)?/?$");
```

Comment:
Why are these patterns changing?

Reply:
Because they are too lenient, they match URLs that are not valid JitPack URLs

Reply:
For example it will match https://github.com/jbangdev/jbang/blob/main/itests/SankeyPlotTest.java , even though that's obviously neither a "tree" nor a "commit".